lib/src/app/player.js

```diff
@@ -86,17 +87,15 @@ class CastPlayer {
                 logger.error(LOG_PREFIX, 'Cannot parse params:', e);
             }
         }
-
-        let params = Object.assign(
-            { playerBrandingId: this.params.playerBrandingId, pcode: this.params.pcode },
-            data.params
+        let params = createPageLevelParameters(
```

Comment:
Why this helper function is used twice on this method?

Reply:
Thank you, fixed